subsys/bluetooth/controller/ll_sw/ull_conn.c

```diff
@@ -3136,14 +3136,16 @@ static inline void event_len_prep(struct ll_conn *conn)
 		struct pdu_data_llctrl_length_req *lr;
 		struct pdu_data *pdu_ctrl_tx;
 		struct node_tx *tx;
+		/*
```

Comment:
there initializations are not needed if you have lines 3233 and 3234

Reply:
They were added because the Clang compiler generated warnings/errors before the changes from this PR were added. No consequence for code-side, so I prefer to not remove this initialisation